subsystems/provisioning/definitions.go

```diff
@@ -29,6 +29,10 @@ const (
 	ConnCheckFilepath = "/etc/NetworkManager/conf.d/80-viam.conf"
 	ConnCheckContents = "[connectivity]\nuri=http://packages.viam.com/check_network_status.txt\ninterval=300\n"
 
+	wifiPowerSaveFilepath        = "/etc/NetworkManager/conf.d/81-viam-wifi-powersave.conf"
```

Comment:
These belonged here with similar definitions for the other files.

Also, changed filename to avoid potential collisions with user-managed/written files.